resources/cassandra/INSTALL.md

```diff
@@ -0,0 +1,57 @@
+# Prerequisites
+The cassandra exporter generates a high number of metrics as the Sysdig Agent has a limit of timeseries that can send to Sysdig Monitor, you have to deploy a Prometheus server and create the recording rules that we provide. This way, we will filter only the metrics that we need.
```

Comment:
The cassandra exporter generates a high number of metrics. As the Sysdig Agent has a limit of time series that can send to Sysdig Monitor, you have to deploy a Prometheus server and create the recording rules that we provide to filter only the metrics used in dashboards and alerts.